examples/look/tests/test_app.py

```diff
@@ -25,19 +28,17 @@ def client(mock_store):
     return testing.TestClient(api)
 
 
-def test_list_images(client):
-    doc = {
-        'images': [
-            {
-                'href': '/images/1eaf6ef1-7f2d-4ecc-a8d5-6e8adba7cc0e.png',
-            },
-        ],
-    }
+def test_list_images(client, mock_store):
+    images = ['first-file', 'second-file', 'third-file']
+    image_docs = [{'href': '/images/' + image} for image in images]
+
+    mock_store.list.return_value = images
 
     response = client.simulate_get('/images')
-    result_doc = msgpack.unpackb(response.content, raw=False)
 
-    assert result_doc == doc
+    result = json.loads(response.content)
```

Comment:
we can use json() here `response.json()`

Reply:
Done